data_steward/admin/key_rotation.py

```diff
@@ -82,7 +82,7 @@ def delete_key(key):
         LOGGER.info(
             '{full_key_name} is deleted'.format(full_key_name=full_key_name))
```

Comment:
shouldn't this also be an f-string?

Reply:
Indeed.. pylint appears to not care about `format` style logs. In some cases I've found it actually may still be necessary to use this style - specifically because you can't inline backslashes in an f-string expression (see further down in the PR). We could perhaps revisit when upgrading pylint. Fixed for now though